### PR TITLE
Kan 110 request documentation

### PR DESCRIPTION
**Pull request by sakarisalmi**
I advice Jenny to check the attribute descriptions to check if I wrote something wrong.
I also changed the private methods to have the '_' at the end. There was a function setCgiFlag(void) which is private, but there is also a public setCgiFlag(bool). I went through the symbols and didn't change the symbol when it was the public function.

I tested the program with the default conf file and everything worked.

### CODE REVIEW

File: include/Request.hpp
Comment (jboucher154) [inline include/Request.hpp:1]: body_size_ is the size promised by the Content-Length header

Comment (jboucher154) [inline include/Request.hpp:1]: What you mean vis-a-vis ?  -> string of the raw body (vis-a-vis processed_body_)

Comment (jboucher154) [inline include/Request.hpp:1]: query_encode_;			/*!< \brief bool that represents if there is a encoded query in the request or not

it indicates if the response should encode or not

File: srcs/Request.cpp
Comment (jboucher154) [inline srcs/Request.cpp:1]: parseBoundary name not changed to ahce `_' at end

**Review by jboucher154 — CHANGES_REQUESTED** [review]
Just a few small things to change, see comments

File: include/Request.hpp
Comment (sakarisalmi) [reply include/Request.hpp:1]: It means 'face-to-face' (as in comparison or opposite of something else)

**Review by jboucher154 — APPROVED** [review]
Thanks for making those changes!

**Pull request merged**